cmake/FindXSMM.cmake

```diff
@@ -0,0 +1,36 @@
+# - Find the XSMM library
```

Comment:
Are we using this file anywhere?

Reply:
CMakeLists.txt, line 359

Reply:
Oh how cool. I didn't realize that you could extend CMake that way. Thanks!